app/logic/pager.rb

```diff
@@ -0,0 +1,18 @@
+class Pager
+
+  def info
+    { per_page: per_page, on_page: on_page }
+  end
+
+  private
+
+  def per_page
+    List.per_page
+  end
+
+  def on_page
+    total = List.count
+    total.div(per_page) + 1
+  end
```

Comment:
@vlasiak, moving to a view object pass `amount` parameter into constructor (`List.count` from controller).